rsyslog/Dockerfile

```diff
@@ -24,12 +24,14 @@ FROM registry.svc.ci.openshift.org/ocp/4.0:base
 
 COPY --from=builder /RPMS/ /RPMS/
 
-RUN yum -y install /RPMS/*.rpm && \
+RUN yum -y install /RPMS/*.rpm cronie && \
     rm -rf /RPMS && \
     yum clean all
 
 ADD install.sh /bin/install.sh
 ADD rsyslog.sh /bin/rsyslog.sh
 ADD uninstall.sh /bin/uninstall.sh
+COPY utils/** /usr/local/bin/
+RUN sed -i -e 's,\(session[ 	]*required[ 	]*pam_loginuid.so\),\#\1,' /etc/pam.d/crond
```

Comment:
We need this when the container is used for cron/logrotate container, but does it have any effect when the container is used for rsyslog?

Reply:
Yes, that is correct.  How can I add it just to the logrotate container?

Reply:
>  How can I add it just to the logrotate container?

Not sure, but I wouldn't worry about it unless it causes problems for the rsyslog container.

Reply:
An interesting thing is to rsh into the container, you could do `"oc rsh $RSYSLOG_POD -c rsyslog"` for rsyslog, but it does not work for logrotate.  I.e., `"oc rsh $RSYSLOG_POD -c logrotate"` fails.  But kubectl works `"kubectl exec -it $RSYSLOG_POD -c logrotate /bin/sh"`.

Do you think it's normal?

Reply:
I don't know, but it doesn't matter as long as rsyslog works normally as an openshift pod.